engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1403,7 +1403,7 @@ void paladin_t::init_base_stats()
   base.spell_power_per_intellect = 1.0;
 
   // Boundless Conviction raises max holy power to 5
-  resources.base[ RESOURCE_HOLY_POWER ] = 3 + passives.boundless_conviction -> effectN( 1 ).base_value();
+  resources.base[ RESOURCE_HOLY_POWER ] = resources.max[ RESOURCE_HOLY_POWER ] = 3 + passives.boundless_conviction -> effectN( 1 ).base_value();
```

Comment:
resources.max shouldn't need to be set here.

Reply:
It was set so it could be used in https://github.com/simulationcraft/simc/pull/5411#discussion_r505546437, but after restructuring the flow didn't even need it.